tests/arp/test_stress_arp.py

```diff
@@ -96,15 +99,15 @@ def test_ipv4_arp(duthost, garp_enabled, ip_and_intf_info, intfs_for_test,
     if normalized_level is None:
         normalized_level = "debug"
 
-    ipv4_avaliable = get_crm_resources(duthost, "ipv4_neighbor", "available")
-    fdb_avaliable = get_crm_resources(duthost, "fdb_entry", "available")
-    pytest_assert(ipv4_avaliable > 0 and fdb_avaliable > 0, "Entries have been filled")
+    ipv4_available = get_crm_resources(duthost, "ipv4_neighbor", "available")
```

Comment:
What changed here? Please make sure there are no additional whitespaces introduced in this line and other lines below.

Reply:
I just replaced a typo and replaced all "avaliable" to "available"